_config.yml

```diff
@@ -18,8 +18,8 @@ description:
   en: Working in the open! Our work in progress, list of reference material and presentations.
   fr: Travailler ouvertement! Nos travaux en cours, liste de documents de références et présentations.
 sectionsList:
-  en: ["About us", "Work In Progress", "IT Picture", "Trends", "Under Review", "Archives"]
-  fr: ["À propos de nous", "Travaux en cours", "Image de la TI", "Tendances", "En révision", "Archives"]
+  en: ["About us","IT Picture", "Trends", "Under Review", "Work In Progress", "Archives"]
```

Comment:
Instead of "IT Picture", I would use "Published Documents"

We may want to also have a versioning mechanism to display when published documents change over time. So that if a reader comes back, she knows something has changed.

Having both a "Published Documents" and "Work in Progress" sections would mean:
- Any changes or work on "Published Documents" would need to be in branches. So if a reader wants to know what we are working on to change a published document would require that reader to know about GitHub and the branches.
- "Work in Progress" are new documents that are not yet published but can still be merged in master so that they are displayed on our site.

Reply:
The Published Documents should also have a section about "Sign off" with links to the approval (I suspect just our DG and our CIO?)
Such links may be to our internal Sharepoint (e.g. emails saying "approved" or the very digital PDF version with signature)

Reply:
Yes, I can make the first change but for the more advanced parts, please create an issue so we work on it in a following PR. Thanks!

Reply:
I like the idea but "Published Documents" sounds a little weird as all our documents are actually "published" on the website...

A term that communicates "Good Enough" would be good. Or maybe an indicator of quality or maturity (low/medium/high)?

Reply:
OK, going with Good Enough at the moment. New PR if you want to update to new term.